js/main.js

```diff
@@ -36,4 +41,71 @@ function handleInputs(event) {
   data.entries.push(objectOfValues);
   $displayedImage.setAttribute('src', 'images/placeholder-image-square.jpg');
   $form.reset();
+
+  $dataViewEntries.classList.remove('hidden');
+  $dataViewEntryForm.classList.add('hidden');
+  data.view = 'entries';
+  location.reload();
 }
+
+window.addEventListener('DOMContentLoaded', addAnEntry);
+function addAnEntry(entry) {
+  var ulListEntries = document.createElement('ul');
+  ulListEntries.setAttribute('class', 'list-entries');
+  if (data.entries.length > 0) {
+    for (var i = 0; i < data.entries.length; i++) {
+      var liRowListItem = document.createElement('li');
```

Comment:
so one thing you want to do is have the function to render the `lis` as a separate render function, like in the javascript dom creation exercise. The loop should call the function, pass in the entry, then append the returning entry. you want to make the all the function does is create the DOM tree and return it, you don't want to append it in the function.